javascript.js

```diff
@@ -1,13 +1,31 @@
-let boardInteger = 3;
+let boardSize = 3;
```

Comment:
I changed the name of this variable bc it's not great to have the variable type in the variable name. (i.e. it should be obvious through your usage of the variable that it's an integer)